wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/SubsystemBase.java

```diff
@@ -15,7 +15,9 @@
  * <p>This class is provided by the NewCommands VendorDep
  */
 public abstract class SubsystemBase implements Subsystem, Sendable {
-  /** Constructor. */
+  /**
+   * Constructor.  Telemetry/log name defaults to the classname.
+   */
```

Comment:
This constructor should delegate to the added one.

Reply:
Can't because of constructor semantics.